common/middleware/countries/countries.tests.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+// npm dependencies
+const expect = require('chai').expect
+
+// Local dependencies
+const countries = require('./index')
+
+describe('countries middleware', function () {
```

Comment:
`countries middleware` --> `countries util`

Reply:
Addressed in [54d67b2]